sdk/cmake/config.cmake

```diff
@@ -40,7 +40,12 @@ if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
     set(GCC TRUE CACHE BOOL "The compiler is GCC")
     set(CLANG FALSE CACHE BOOL "The compiler is LLVM Clang")
 elseif(CMAKE_C_COMPILER_ID STREQUAL "Clang")
-    set(GCC FALSE CACHE BOOL "The compiler is GCC")
+    # We can use LLVM Clang mimicking CL or GCC. Account for this
+    if (MSVC)
+        set(GCC FALSE CACHE BOOL "The compiler is GCC")
```

Comment:
I have a feeling this would complicate things too much. Is there any better solution?

Reply:
Not setting GCC would complicate things more. As clang-cl does with MSVC, clang aims to be CLI-compatible with GCC.

While building, I didn't notice much difference there.